src/search/utils.py

```diff
@@ -24,3 +27,33 @@ def clean_search_querystring(querystring):
             querydict_cleaned.update(MultiValueDict(dict_cleaned))
             querystring = querydict_cleaned.urlencode()
     return querystring
+
+
+def get_querystring_value_list_from_key(querystring, key):
+    querydict = QueryDict(querystring)
+    value_list = querydict.getlist(key)
+    value_list_cleaned = list(filter(None, value_list))
+    return value_list_cleaned
+
+
+def get_querystring_perimeter(querystring):
+    PERIMETER_KEY = 'perimeter'
+    perimeter_list = get_querystring_value_list_from_key(querystring, PERIMETER_KEY)  # noqa
+    if len(perimeter_list):
+        # get first perimeter id
```

Comment:
Proposition de commentaire :

We expect only one perimeter and it's prefixed with the ID, ex : "71045-rhone"